app/styles/globals.css

```diff
@@ -10,6 +10,10 @@
   -webkit-font-smoothing: antialiased;
 }
 
+ion-icon {
+  pointer-events: none;
```

Comment:
I think we need to find a different solution than this. This removes the hover-effect from icons, and makes cypress refuse to click any icons.

Reply:
I just removed this then :/ I couldn't really find another good way to get rid of it, and it's not _that_ important.